test/check-machines-lifecycle

```diff
@@ -420,6 +420,16 @@ class TestMachinesLifecycle(VirtualMachinesCase):
             qemu_img_info = m.execute(f"qemu-img info /var/lib/libvirt/images/{name}-2.img")
             self.assertNotIn("snapshotA", qemu_img_info)
 
+        # Try to delete a VM and keep its volume
+        name = "vm-keep-vol"
+        args = self.createVm(name)
+        self.waitVmRow(name)
+        self.performAction(name, "delete")
+        b.set_checked("#vm-{}-delete-modal-dialog input[type=checkbox]".format(name), False)
+        b.click("#vm-{}-delete-modal-dialog button:contains(Delete)".format(name))
+        self.waitVmRow(name, present=False)
+        wait(lambda: m.execute("test -f {} && echo $?".format(args["image"])))
```

Comment:
Thanks, this looks good. One comment is that we are trying to phase out "{}".format(var) for a f"{var}", which is not used almost everywhere in this file. 
```suggestion
        # Try to delete a VM and keep its volume
        name = "vm-keep-vol"
        args = self.createVm(name)
        self.waitVmRow(name)
        self.performAction(name, "delete")
        b.set_checked(f"#vm-{name}-delete-modal-dialog input[type=checkbox]", False)
        b.click("#vm-{name}-delete-modal-dialog button:contains(Delete)")
        self.waitVmRow(name, present=False)
        wait(lambda: m.execute(f"test -f {args["image"]} && echo $?"))
```